testData/inspections/php/GraphQlResolverInspection/withValidResolverInterface/ValidResolverTest.php

```diff
@@ -4,7 +4,7 @@
  * See COPYING.txt for license details.
  */
 namespace Magento\Test;
-class ValidResolverTest implements \Magento\Framework\GraphQl\Query\ResolverInterface
+class ResolverTest implements \Magento\Framework\GraphQl\Query\ResolverInterface
```

Comment:
Please make the file name and class name same
Thanks

Reply:
Done, thank you!